Fix nod05 airstrike Lua crash and update map rules

Biofreak1987 opened the pull request:
Don't call an airstrike while the player has no ground units.

Phrohdoh commented:
You could just bail out of the function instead of wrapping its entire purpose in an if.

Biofreak1987 commented:
Updated

obrakmann commented:
:+1:

Biofreak1987 commented:
Actually... I just notice that nod05 also shows the owner row on husks. Should I fix that with this pr as well or create another one?

obrakmann commented:
You can do it here

Commit:
Fix nod05 airstrike Lua crash and update map rules

- Fix airstrike Lua crash when there are no targets
- Increased harvester search range
- Hide owner row from husks and walls

Changes to make it closer to the original:
- Added e3 and sandbag production
- Made airstrikes gdi only
- Limited airstrike squadsize to 1
- Allow to attack civilian buildings

Biofreak1987 commented:
K, added

Phrohdoh commented:
The commit message no longer covers everything changed in this single commit.

Biofreak1987 commented:
Sorry, fixed that.

Biofreak1987 commented:
By replaying the original mission I've noticed that there are a few other things missing (you can build e3 units and sand bags in the original)... also nod can't use airstrikes in their campaign, should that be then also the case for the ports?

Biofreak1987 commented:
I've ended up adding those changes so that the map ends up being closer to the original version. Details about it are in the commit message.

obrakmann commented:
FWIW, my :+1: still stands after the additional changes.

abcdefg30 commented:
Looks good to me, too. :+1: / :white_check_mark:

abcdefg30 commented:
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/6eeb35be7b58c73b84fd3db5cda2fc75c469bfd2...00ad73a6128e3477dea65aac93c4104b3a55ec8a)
